torch/_inductor/compile_fx.py

```diff
@@ -889,39 +917,41 @@ def wrapper(args):
         args.clear()
         return optimized_function(args_new)
 
-    wrapper._boxed_call = True
+    wrapper._boxed_call = True  # type: ignore[attr-defined]
 
     return wrapper
 
 
 def compile_fx(
     model_: torch.fx.GraphModule,
     example_inputs_: List[torch.Tensor],
-    inner_compile=compile_fx_inner,
+    inner_compile: Callable[..., Any] = compile_fx_inner,
     config_patches: Optional[Dict[str, Any]] = None,
-    decompositions: Optional[Dict[OpOverload, Callable]] = None,
+    decompositions: Optional[Dict[OpOverload, Callable[..., Any]]] = None,
 ):
     """Main entrypoint to a compile given FX graph"""
     if config_patches:
-        with config.patch(config_patches):
+        with config.patch(config_patches):  # type: ignore[attr-defined]
```

Comment:
same question here, what is the attr-defined error ? given this is a common idiom, wonder if theres a way of fixing this at source.

Reply:
Fixed. Actually, removing these type ignores didn't trigger any mypy errors.